packages/utils/__tests__/is-local-path.test.ts

```diff
@@ -1,7 +1,7 @@
 "use strict";
 
 import * as path from "path";
-import isLocalPath from "../is-local-path";
+import { isLocalPath } from "../path-utils";
```

Comment:
absolute import by package here, to avoid breaking stuff

Reply:
Hey @evenstensberg if you mean importing by the package name like this - 
![image](https://user-images.githubusercontent.com/21009455/56507530-38b14f00-653f-11e9-9f44-a000596042a7.png) this doesn't work clearly even after symlink (same issue as before) and if you intend to do something like `import x from 'utils/path-utils` then that would need changes to the tsconfig file to add entries like `baseUrl` and `paths` and stuff which I don't think needs to be done here since tests seem to work fine both locally and on CI.

Or maybe you can suggest something? :)

Reply:
I think you'll need to change how the CI bootstraps in that case. You should get it up running locally first.

Reply:
@evenstensberg I investigated on this, everything works if I do 
```sh
cd packages/utils
npm link
npm link @webpack-cli/utils
```
which basically creates a symlink of the utils package which should originally be done by lerna.
There seems to be some issue with lerna https://github.com/webpack/webpack-cli/pull/806#issuecomment-480594467 and thus I'm still figuring this out since I started working on this. Can we go ahead with this meanwhile?

Reply:
Literally _all_ of the sibling tests in this module use relative imports of the file under test. Really unclear how using absolute package imports will help "avoid breaking stuff."

Reply:
cc @evenstensberg 

Thanks @evocateur 🙏